### PR TITLE
feat: Add example and support for new loss mode

### PR DESCRIPTION
**Pull request by shahrukhqasim**
Still have to modify plotting scripts etc.

### CODE REVIEW

**Comment by jkiesele** [comment]
Thanks for the PR.
It looks good in general, I just have a few comments:
a) I think you misunderstood the aim of the LLClusterCoordinates layer. This layer is meant to help GravNet find good coordinates, it is not meant to do full object condensation. For this you would need to add another new layer.
b) Once you did (a), please make sure (also with numerical tests) that the new layer (name it e.g. "LLObjectCondensation") does the same as the functions before.

**Comment by shahrukhqasim** [comment]
Okay. It should be simple enough to do. On it.

File: modules/lossLayers.py [hunk 1]
```diff
@@ -59,4 +63,111 @@ def loss(self, inputs):
...
+        self.huber_energy_scale = huber_energy_scale
+        self.downweight_low_energy = downweight_low_energy
+        self.n_ccoords = n_ccoords
+        self.energy_den_offset = energy_den_offset
+        self.noise_scaler = noise_scaler
+        self.too_much_beta_scale = too_much_beta_scale
+        self.cont_beta_loss = cont_beta_loss
+        self.log_energy = log_energy
+
+        if standard_configutation is not None:
```
Comment (jkiesele) [inline modules/lossLayers.py:108]: just a typo

Comment (shahrukhqasim) [reply modules/lossLayers.py]: fixed.

File: modules/betaLosses.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def __init__(self):
         self.log_energy=False
 
 
-config = _obj_cond_config()
+config = construct_obj_cond_config()
```
Comment (jkiesele) [inline modules/betaLosses.py:39]: is this still necessary?

Comment (shahrukhqasim) [reply modules/betaLosses.py:39]: not anymore let me remove it

Comment (shahrukhqasim) [reply modules/betaLosses.py]: done

File: modules/lossLayers.py [hunk 3]
```diff
@@ -35,24 +36,139 @@ def loss(self, inputs):
...
+            'n_ccoords': self.n_ccoords,
+            'energy_den_offset': self.energy_den_offset,
+            'noise_scaler': self.noise_scaler,
+            'too_much_beta_scale': self.too_much_beta_scale,
+            'cont_beta_loss': self.cont_beta_loss,
+            'log_energy': self.log_energy,
+        }
+
+        loss = obj_cond_loss(truth_dict, pred_dict, feat_dict, row_splits, config)
+        loss = full_obj_cond_loss(truth_dict, pred_dict, feat_dict, row_splits)
```
Comment (jkiesele) [inline modules/lossLayers.py:142]: which one?

Comment (shahrukhqasim) [reply modules/lossLayers.py]: The first one - fixed now. It was to see the numerical results difference and I forgot to remove it.
